lib/ca_certs.ml

```diff
@@ -85,7 +85,15 @@ let trust_anchors () =
     Bos.OS.Cmd.(run_out cmd |> out_string |> success) >>= function
     | "FreeBSD" -> detect_one freebsd_location
     | "OpenBSD" -> detect_one openbsd_location
-    | "Linux" -> detect_list linux_locations
+    | "Linux" ->
+      begin
+        (* NixOS is special and sets "NIX_SSL_CERT_FILE" as location *)
```

Comment:
This should probably clarify that this is only the case for the sandbox used for building nix packages: At run time on normal NixOS `/etc/ssl/certs/{ca-bundle.crt, ca-certificates.crt}` do both exist and `NIX_SSL_CERT_FILE` is *not* set.